conf/messages.en

```diff
@@ -325,7 +325,7 @@ declaration.declarant-details.help.bodyText = Go to data element 3/18 Declarant
 declaration.declarant.eori.empty = Enter your EORI number
 declaration.declarant.eori.error.format = Enter a valid EORI number, for example GB123456789000
 
-supplementary.consignee.title = What is the consignee’s address?
+declaration.consignee.title = What is the consignee’s address?
 supplementary.consignee.title.hint = Enter the consignee’s EORI number or name and business address.
 supplementary.consignee.eori.info = Consignee’s EORI number
 supplementary.consignee.address.info = Consignee’s name and business address
```

Comment:
Can you change other messages related with this page to use `declaration` in message key?

Reply:
The other messages are not actually used. I will delete them.